reports/src/app-links.ts

```diff
@@ -54,7 +54,7 @@ export class AppLinks {
       { type: 'Tab', title: 'Tracks', link:  'clc://clc/app-link/Tracks' },
       { type: 'Tab', title: 'Connect', link:  'clc://clc/app-link/Connect' },
       { type: 'Tab', title: 'Staff Directory', link:  'clc://clc/app-link/StaffDirectory' },
-      { type: 'Tab', title: 'Settings', link:  'clc://clc/app-link/UserSettings' }
```

Comment:
do we no longer want the ability to link to settings?

Reply:
This one wasn't working and I never invested time to debug why